src/gui/qgsattributeeditorcontext.h

```diff
@@ -204,6 +220,27 @@ class GUI_EXPORT QgsAttributeEditorContext
      */
     void setFormFeature( const QgsFeature &feature ) { mFormFeature = feature ; }
 
+    /**
+     * Returns current attributeFormMode
+     * \since QGIS 3.4
+     */
+    Mode attributeFormMode() const { return mAttributeFormMode; }
+
+    /**
+     * Set \a attributeFormMode for the edited form
+     * \since QGIS 3.4
+     */
+    void setAttributeFormMode( const Mode &attributeFormMode ) { mAttributeFormMode = attributeFormMode; }
+
+    /**
+     * Returns given \a attributeFormMode as string
+     * \since QGIS 3.4
+     */
+    QString attributeFormModeString( const Mode &attributeFormMode )
```

Comment:
I think this should either be static, or (preferred) no parameter and work on `mAttributeFormMode` instead.

Reply:
Oh, and const